tools/sm_teledump.lua

```diff
@@ -64,7 +61,7 @@ local MEMTRACK = { -- {{{
     { 0x09CC, 0x2, 'Max supers' },
     { 0x09CE, 0x2, 'Pbs' },
     { 0x09D0, 0x2, 'Max pbs' },
-    -- { 0x09D2, 0x2, 'Current selected weapon' },
+    { 0x09D2, 0x2, 'Currently selected item' },
```

Comment:
I would leave this commented out. It's very common to leave missiles or supers selected when setting up for a preset, and that's generally unwanted because the door will be blue when the preset is loaded. It's easier to add in lines to select power bombs than to delete a bunch of lines to turn off missiles.

Reply:
I've been thinking about saving/loading which doors are open.  I don't believe it is directly tracked by the game so not sure how feasible it is.

Even if it can be done, I'm not sure what's going to happen when you run the lua scripts.  Does it play through with the vanilla rom?  The practice rom that they were recorded with?  Or can the latest practice rom be used with the previously recorded inputs?  That'd be the only way door tracking could be added to the lua scripts and have it actually work.

Also if you have any head start on door open stuff that might be useful.  iirc you looked into once a while back.

Reply:
I have no progress on that, but here's the chat I had with Smiley and PJ about it.
https://discord.com/channels/127475613073145858/370769325851803648/867761505872642048

Reply:
The emulator won't let you play back a movie from a different ROM than it was made with.

Reply:
Oh duh, instead of a new variable determining which doors to open, I should go with your idea to make all of the doors open (or at least all of the blue doors).  That's simpler and no need to worry about lua.

My plan was to search the PLMs but I see from the discussion that's not going to work.  Searching the BTS for 40-43 would probably work though.  I'll look into that.